mdm/internal/checkinproto/checkin.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.18.1
+// 	protoc-gen-go v1.28.0
+// 	protoc        v3.15.8
```

Comment:
Go protobuf was updated but it looks like you're running an older version of protoc. Would you mind upgrading that?

Reply:
[upgrade protoc to 3.21.2](https://github.com/micromdm/micromdm/pull/839/commits/2dc7b8d5c51b7a7f6d6f06b778c2ba069f045c33)

upgraded to latest protoc